src/Http/Controllers/SwaggerController.php

```diff
@@ -33,8 +38,15 @@ public function docs($jsonFile = null)
 
         $content = File::get($filePath);
 
+        $contentType = 'application/json';
+        if ($extension === 'yaml') {
```

Comment:
Method description sugests `yml or yaml file` but only `yaml` extension is checked

Reply:
I'll update the docs. I was going to correct .yml to look for the *.yaml file, but that seemed to complex for the simple use case.